src/fermilib/transforms/_bksf_test.py

```diff
@@ -0,0 +1,194 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""Tests  _bravyi_kitaev_fast_test.py."""
+from __future__ import absolute_import
```

Comment:
reorder imports into correct form

1. Python modules
2. External modules
3. FermiLib stuff
4. the implemenation file